packages/dart/lib/src/objects/parse_user.dart

```diff
@@ -315,7 +315,16 @@ class ParseUser extends ParseObject implements ParseCloneable {
   /// server. Will also delete the local user data unless
   /// deleteLocalUserData is false.
   Future<ParseResponse> logout({bool deleteLocalUserData = true}) async {
-    final String sessionId = ParseCoreData().sessionId!;
+    final String? sessionId = ParseCoreData().sessionId;
+
```

Comment:
Explaining: I found this bug before the package was transferred to Parse. When calling the logout function without having previously signed or sigup, a null error is displayed.

At that time we didn't have the routine of creating a bug for every fix in the code.

Reply:
Could you please make sure this fix has a changelog entry in this PR?

Reply:
done